specs.json

```diff
@@ -698,6 +699,25 @@
     "required": [
     ]
   },
+  "googleOptions" : {
+    "properties": {
+      "serviceVersion": {
+        "type": "string",
+        "enum": [
+          "v1",
+          "v2"
+        ]
+      },
+      "voiceActivityTimeout": {
+        "type": "string",
+        "enum": [
+          "speech_start_timeout",
+          "speech_end_timeout"
+        ]
+      },
+      "transcriptNormalization": "array"
```

Comment:
Yes this is good.  As far as what the user will provide it should actually be an array of arrays, e.g
```
[['dog', 'cat'], ['damn it', 'darn it']]
```

Reply:
yes